### DISCUSSION TITLE
Github Organization Pro fro student tema

### DISCUSSION THREAD

**Discussion by yahiaz7** [community/community#147463] [GitHub Education]
### Hi everyone,

Hi everyone, 
Is there a Plus version for GitHub organization for a student team. I have seen a reply on previous disccsion before (2022) but is not working anymore. 

Note: everyone has a s plus student  account

**Comment by Aditya7615** [comment]
While the GitHub Student Developer Pack provides individual students with a free GitHub Pro account, a student team or organization can get a free GitHub Team account for advanced features through a verified teacher or faculty member. The teacher applies for the benefit, which then grants the team access to features like unlimited private repositories and enhanced collaboration tools.
